Libplanet.Tools/Mpt.cs

```diff
@@ -222,5 +194,29 @@ private IKeyValueStore LoadKVStoreFromURI(string rawUri)
 
             return constructor(uri.AbsolutePath);
         }
+
+        private string ConvertKVStoreUri(string kvStoreUri, ToolConfiguration toolConfiguration)
+        {
+            // If it is not Uri format,
+            // try to get uri from configuration service by using it as alias.
+            if (!Uri.IsWellFormedUriString(kvStoreUri, UriKind.Absolute))
```

Comment:
It would be better if we make `planet mpt add` command to disallow the alias to be a well-formed URI.

Reply:
Yes, exactly.

Reply:
I applied it in 9a91042

It will work like ![image](https://user-images.githubusercontent.com/26626194/102069055-985d1c00-3e40-11eb-9f2d-001c742e76bf.png)